README.md

```diff
@@ -1 +1,200 @@
+      }
+    },
+    {
+      name = "data-disk-0"
+
+      gcePersistentDisk = {
+        pdName = "data-disk-0"
+        fsType = "ext4"
+      }
+    },
+  ]
+
+  restart_policy = "Always"
+}
+```
+
+Then perform the following commands on the root folder:
+
+- `terraform init` to get the plugins
+- `terraform plan` to see the infrastructure plan
+- `terraform apply` to apply the infrastructure build
+- `terraform destroy` to destroy the built infrastructure
+[^]: (autogen_docs_start)
+
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| container | A description of the container to deploy | map | `<map>` | no |
+| restart_policy | The restart policy for a Docker container. Defaults to `OnFailure` | string | `OnFailure` | no |
+| volumes | A set of Docker Volumes to configure | list | `<list>` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| container |  |
+| metadata_key | The key to assign `metadata_value` to, so container information is attached to the instance |
+| metadata_value | The generated container configuration |
+| restart_policy |  |
+| source_image | The COS image to use for the GCE instance |
+| vm_container_label | The COS version to deploy to the instance. To be used as the value for the `vm_container_label_key` label key |
+| vm_container_label_key | The label key for the COS version deployed to the instance |
+| volumes |  |
+
+[^]: (autogen_docs_end)
+
+## Requirements
+### Terraform plugins
+- [Terraform](https://www.terraform.io/downloads.html) 0.10.x
+- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v1.8.0
+
+### Python Libraries
+- [terraform_external_data](https://github.com/operatingops/terraform_external_data)
+
+### Configure a Service Account
+In order to execute this module you must have a Service Account with the following:
+
+#### Permissions
+- `compute.disks.*` on the project
+- `compute.diskTypes.get` on the project
+- `compute.diskTypes.list` on the project
+
+### Enable API's
+In order to operate with the Service Account you must activate the following APIs on the project where the Service Account was created:
+
+- Compute Engine API - compute.googleapis.com
+
+## Install
+
+### Terraform
+Be sure you have the correct Terraform version (0.10.x), you can choose the binary here:
+- https://releases.hashicorp.com/terraform/
+
+## File structure
+The project has the following folders and files:
+
+- /: root folder
+- /examples: examples for using this module
+- /helpers: Scripts that the module invokes
+- /test: Folders with files for testing the module (see Testing section of this file)
+- /main.tf: main file for this module, contains all the resources to create
+- /variables.tf: all the variables for the module
+- /output.tf: the outputs of the module
+- /readme.MD: this file
+
+## Testing
+
+### Requirements
+- [bundler](https://github.com/bundler/bundler)
```

Comment:
Should we be listing `docker` as a dependency?

Reply:
Good catch -- I'll tweak the README to reflect how Docker is used for integration testing.